tools/templates/js/Makefile.in

```diff
@@ -22,6 +22,8 @@
 	@echo(!!! Installing the js backend is not yet implemented.)@
 	$(NOECHO)exit 1
 
+@bpv(BACKEND_SPECIFIC_TEST_FILES)@ = t/11-js
```

Comment:
`nfp` again. But I wonder what does it supposed to do in first place?

Reply:
I'm not sure, I understand your question. Does it refer to introducing a new Makefile variable or to the use of ```@bpv()@```?

I thought it would make sense to declare the backend specific test dirs (sic!) in the backend specific Makefile templates. So a new variable looked logical to me.

Wrt ```@bpv()@```: The macro stuff looked scary to me, so I tried to cargo cult from the surrounding code. (I probably should have looked at ```NQP::Macros.pm```.) With my newly pushed commit I've changed to ```@bsv()@```, because we use that for ```METAMODEL_SOURCES``` (which looks structurally similar to the test dirs).

Reply:
I'm sorry for the totally legit confusion of yours and recall my question. I overlooked the fact that the "duplicated" lines are from different backend templates.